src/User/Login.js

```diff
@@ -1,56 +1,54 @@
 import React from 'react'
-import {connect} from 'react-redux'
+import { connect } from 'react-redux'
 import _ from 'lodash/fp'
-import image from './login.jpg'
-import {Link, Redirect} from 'react-router-dom'
-import {withTranslation} from 'react-i18next'
+import backgroundImage from './loginImage.jpg'
+import backgroundImageMobil from './loginImageMobil.jpg'
```

Comment:
die jpgs bitte zu den anderen Bildern in _src/images_ verschieben.

Reply:
Ich hab sie hier nicht gebracht, sie waren schon hier.
aber du hast recht, ich habe sie in Images Folder verleget